lib/whitehall/authority/rules/document_rules.rb

```diff
@@ -8,7 +8,12 @@ def initialize(actor, subject)
     end
 
     def can?(action)
-      actor.gds_editor? || actor.departmental_editor? || action == :create
+      case action
+      when :perform_administrative_tasks
+        actor.gds_editor? ? true : false
```

Comment:
Do we need the ternary operator here?

Reply:
Thanks for the review. Yea good point! Will sort it 👍